test/utils/zip.ts

```diff
@@ -0,0 +1,45 @@
+import { createHash } from "crypto";
+import { get } from ".";
+const AdmZip = require("adm-zip");
+
+/**
+ * Gets the list of files in a zip file located at the specified URL
+ * @param {string} url - The URL of the zip file
+ * @returns {Promise<string[]>} - A promise that resolves with the list of files in the zip file
+ */
+export async function getZipFileList(url: string): Promise<string[]> {
+  const zipFileBuffer = await get(url);
+  const zip = new AdmZip(zipFileBuffer);
+  const entries = zip.getEntries();
+  return entries.map((entry) => entry.name);
+}
+
+/**
+ * Calculates the checksum of a file
+ * @param {string} url - The URL of the file to calculate the checksum for
+ * @returns {Promise<string>} - A promise that resolves with the checksum of the file
+ */
+
+export async function calculateChecksum(url: string): Promise<string> {
```

Comment:
this function seems not to be used elsewhere than in the file, no need to export

```suggestion
async function calculateChecksum(url: string): Promise<string> {
```